### PR TITLE
feat(cmd): add .remoteefb command

### PR DESCRIPTION
**Pull request by Lauvaan**
## Description

Fixes: #460 

Adds a command describing the situation of remote acces of the efb, more info in #460 

## Test Results

image

## Discord Username

Lauvaan#3330

### CODE REVIEW

File: .github/command-docs.md [hunk 1]
```diff
@@ -79,7 +79,8 @@
...
-| .reportedissues | Provides a link to the reported issues page within docs, either a general link, or a link for a specific search <br> Search terms can be terminated by a line break or one of the symbols . - > / \                                                                         | .issues                                            |
+| .reportedissues | Provides a link to the reported issues page within docs, either a general link, or a link for a specific search <br> Search terms can be terminated by a line break or one of the symbols . - > / \                                                                       | .issues                                           |            
+| .remoteefb            | Provides information about to the viewing the EFB remotely                                   | .refb <br> .remoteflypad <br> .rflypad           |
```
Comment (pdellaert) [inline .github/command-docs.md:83]: Your `|` aren't lining up with the others, which you should (use spaces, not tabs, in your editor, just to be save)

File: src/commands/support/remoteEfb.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { MessageCommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+const remoteEfbEmbed = makeEmbed({
+    title: 'FlyByWire Support | Remote EFB',
+    description: makeLines([
+        'Currently the EFB cannot be viewed remotely, but it is a planned feature.',
+        '',
+        'In the meantime, you can however use an application like [Spacedesk](https://www.spacedesk.net/)* to create a virtual screen using yout tablet/phone/pc, and then de-attach the EFB from MSFS by holding the ALTGR button and clicking on the EFB with your mouse. You can then drag the EFB window to your virtual screen. Please note that this solution will degrade your performance slightly, as your computer needs to render and process the new virtual screen.',
```
Comment (pdellaert) [inline src/commands/support/remoteEfb.ts:10]:
```suggestion
        'In the meantime, you can however use an application like [Spacedesk](https://www.spacedesk.net/)* to create a virtual screen using your tablet/phone/pc, and then de-attach the EFB from MSFS by holding the ALTGR button and clicking on the EFB with your mouse. You can then drag the EFB window to your virtual screen. Please note that this solution will degrade your performance slightly, as your computer needs to render and process the new virtual screen.',
```
Small typo correction.

**Review by pdellaert — CHANGES_REQUESTED** [review]
Two small changes to look into, otherwise looks good (have not tested yet, but I'm fairly sure it'll be fine)

Needs a CHANGELOG entry too, so that'll block the merge until provided.

File: src/commands/index.ts [hunk 3]
```diff
@@ -151,6 +151,7 @@ import { holds } from './aircraft/holds';
 import { mcdu } from './aircraft/mcdu';
 import { fsltl } from './general/fsltl';
 import { flyPadAbout } from './support/flyPadAbout';
+import { remoteEfb } from './support/remoteEfb';
```
Comment (benw202) [inline src/commands/index.ts:154]: This should go at the end of the list

File: src/commands/index.ts [hunk 4]
```diff
@@ -319,6 +320,7 @@ const commands: BaseCommandDefinition[] = [
     mcdu,
     fsltl,
     flyPadAbout,
+    remoteEfb,
```
Comment (benw202) [inline src/commands/index.ts:323]: This should go at the end of the list

**Review by benw202 — APPROVED** [review]

**Pull request merged**